Doc/howto/clinic.rst

```diff
@@ -1267,16 +1268,17 @@ module state::
 Each method may only have one argument using this converter, and it must appear
 after ``self``, or, if ``self`` is not used, as the first argument.  The argument
 will be of type ``PyTypeObject *``.  The argument will not appear in the
-``__text_signature__``.
+:py:attr:`!__text_signature__`.
 
-The ``defining_class`` converter is not compatible with ``__init__`` and ``__new__``
-methods, which cannot use the ``METH_METHOD`` convention.
+The ``defining_class`` converter is not compatible with
+:py:meth:`!__init__` and :py:meth:`!__new__`
+methods, which cannot use the :c:data:`METH_METHOD` convention.
```

Comment:
```suggestion
methods, which cannot use the :c:data:`!METH_METHOD` convention.
```
I guess this needs a `!` to silence the warning, unless it's an actual target and the warning can be fixed in some other way.

Reply:
I think that the right fix of this issue is to add a reference target for METH_METHOD, so this reference would work. Adding `!` to silence a warning will also prevent rendering a reference when a target be fixed.

There is a declaration
```rst
.. data:: METH_METHOD | METH_FASTCALL | METH_KEYWORDS
```
in Doc/c-api/structures.rst, and it is the only definition of METH_METHOD. Can we add ```.. data:: METH_METHOD``` hidden from the output?

Reply:
I agree with Serhiy; we should fix the `METH_METHOD` target first. I'll get to it.

Reply:
I am working on this. It is complicated.

Reply:
For now, I've disabled the `METH_METHOD` link: b155e3862df036ca4559290e35844db70e2616b9

Reply:
Resolved with 1a30f3b661622d07ff29b0aafc8dda4546c5a999